### PR TITLE
master-AVR: Scripts build system updated.

### PR DESCRIPTION
**Pull request by hiro2233**
Travis and Semaphoreci scripts build udpated and fixed only for apm1 apm2 and sitl.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (gmorph) [inline .travis.yml]: Do we need to tie down the distribution?  We don't do it in the master branch.
Or are you trying to use sudo enabled Ubuntu Trusty for some reason?

File: Tools/scripts/install-travis-env.sh [hunk 2]
```diff
@@ -8,33 +8,21 @@ CWD=$(pwd)
...
-RPI_TARBALL="$RPI_ROOT.tar.gz"
-RPI_TARBALL_URL="http://firmware.ardupilot.org/Tools/Travis/NavIO/$RPI_TARBALL"
+UBUNTU64_PKGS="libc6-i386"
 
 # Ardupilot Tools
 ARDUPILOT_TOOLS="ardupilot/Tools/autotest"
 
 APT_GET="sudo apt-get -qq --assume-yes"
 
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```
Comment (gmorph) [inline Tools/scripts/install-travis-env.sh]: Are we doing this in place of rsyncing or packages below?

File: Tools/scripts/install-travis-env.sh [hunk 3]
```diff
@@ -46,60 +34,18 @@ for pkg in $PYTHON_PKGS; do
     sudo pip -q install $pkg || echo "FAILED INSTALL OF $pkg"
 done
 
-# install some extra packages (for later AVR compiler)
-rsync -av firmware.ardupilot.org::Tools/Travis/*.deb ExtraPackages
-sudo dpkg -i ExtraPackages/*.deb || echo "FAILED INSTALL OF EXTRA DEBS"
-
```
Comment (gmorph) [inline Tools/scripts/install-travis-env.sh]: I think we still need this code to download the correct packages and install them.  See comment above. 

I agree we can/should update the gcc-avr deb package on there to 4.9

File: Tools/scripts/configure-ci.sh [hunk 4]
```diff
@@ -3,16 +3,10 @@
...
       python-argparse python-empy python-serial zlib1g-dev gcc-4.9 g++-4.9"
-	  
-AVR_PKGS="gcc-avr binutils-avr avr-libc"
-
-ARM_ROOT="gcc-arm-none-eabi-4_9-2015q3"
-ARM_TARBALL="$ARM_ROOT-20150921-linux.tar.bz2"
 
-RPI_ROOT="master"
-RPI_TARBALL="$RPI_ROOT.tar.gz"
+AVR_PKGS="gcc-avr binutils-avr avr-libc"
```
Comment (gmorph) [inline Tools/scripts/configure-ci.sh]: This confused me as you deleted it and then put it back in

**Comment by gmorph** [comment]
Overall I'm happy if we want to remove the non AVR build mechanisms on this branch although I'm in the camp if it ain't broke don't fix it.  See my comments about the tooling installation mechanism though.
Thanks.

File: .travis.yml (hunk 1 above)
Comment (hiro2233) [inline .travis.yml]: We are having a conection refuse using rsync from server to update avr libc in precise distribution, thats is the only one reason.

File: Tools/scripts/install-travis-env.sh (hunk 3 above)
Comment (hiro2233) [inline Tools/scripts/install-travis-env.sh]: Yes we need update gcc-avr deb package, please we need grant the server to use rsync, we are getting a conection refusing from server.

File: Tools/scripts/configure-ci.sh (hunk 4 above)
Comment (hiro2233) [inline Tools/scripts/configure-ci.sh]: Yes, i will fix it. Sorry.

**Comment by hiro2233** [comment]
Please see above comments regarding the server connection using rsync, server is refusing the connection. Thanks in advance.

**Comment by tridge** [comment]
pushed, thanks!

**Pull request closed without merging**